pkgs/by-name/ap/apt-mirror/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, perl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apt-mirror";
+  version = "0.5.4";
+
+  src = fetchFromGitHub {
+    owner = "apt-mirror";
+    repo = "apt-mirror";
+    rev = finalAttrs.version;
+    hash = "sha256-GNsyXP/O56Y+8QhoSfMm+ig5lK/K3Cm085jxRt9ZRmI=";
+  };
+
+  nativeBuildInputs = [ perl ];
+
+  installPhase = ''
+    runHook preInstall
+
+    # These commands are taken from the makefile
+    mkdir -p $out/{bin,share/man/man1,etc,var/spool/apt-mirror}
+    install -m 755 apt-mirror $out/bin/apt-mirror
+    pod2man apt-mirror > $out/share/man/man1/apt-mirror.1
+    install -m 644 mirror.list $out/etc/mirror.list
+    mkdir -p $out/var/spool/apt-mirror/{mirror,skel,var}
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A small tool that provides ability to mirror any parts (or even all) of Debian and Ubuntu GNU/Linux distributions or any other apt sources which typically provided by open source developers";
```

Comment:
A shorter description would be nice

Reply:
`"A tool that provides ability to mirror any parts of Debian and Ubuntu distributions or any other apt sources"`

What about this?

Reply:
"Debian-based" is shorter. But your suggestion is nice enough.

Reply:
What about "A tool that provides the ability to mirror any parts of apt sources"?

But your suggestion is nice enough.

Reply:
I liked your suggestion, applied.